unfoldingWord/translationCore
# Feature bspidel fix

#### This pull request addresses:

single change to extend length of text to truncate



#### How to test this pull request:

open project card after import of rmr-x-bsa_mat_ulb from natasha
language should be 2 lines long


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3666)
